src/node_process_methods.cc

```diff
@@ -123,31 +130,27 @@ static void Cwd(const FunctionCallbackInfo<Value>& args) {
   char buf[PATH_MAX_BYTES];
   size_t cwd_len = sizeof(buf);
   int err = uv_cwd(buf, &cwd_len);
-  if (err)
-    return env->ThrowUVException(err, "uv_cwd");
+  if (err) return env->ThrowUVException(err, "uv_cwd");
 
-  Local<String> cwd = String::NewFromUtf8(env->isolate(),
-                                          buf,
-                                          NewStringType::kNormal,
-                                          cwd_len).ToLocalChecked();
+  Local<String> cwd =
+      String::NewFromUtf8(env->isolate(), buf, NewStringType::kNormal, cwd_len)
+          .ToLocalChecked();
   args.GetReturnValue().Set(cwd);
 }
 
 static void Kill(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
   Local<Context> context = env->context();
 
-  if (args.Length() != 2)
-    return env->ThrowError("Bad argument.");
+  if (args.Length() != 2) return env->ThrowError("Bad argument.");
```

Comment:
Do we want these kind of changes?

Reply:
This was done by the clang-formatter 😅 not inentional

Reply:
Can you try reformatting the code and make sure you set `CLANG_FORMAT_START` to where this branch branches off from the master, so that it does not format irrelevant changes? For example since this PR only contains 3 commits you can also try `CLANG_FORMAT_START=HEAD~3 make format-cpp`

Reply:
@joyeecheung Output from the command 

```
❯ CLANG_FORMAT_START=HEAD~3 make format-cpp
Formatting C++ diff from HEAD~3..
clang-format did not modify any files
```

I think what i can do is basically reset to master and do the changes only and do not format the code. In that way those unnecessary changes won't be there.

Reply:
Removed the formatting changes.